src/PrestaShopBundle/Form/Admin/Configure/AdvancedParameters/Administration/NotificationsType.php

```diff
@@ -26,27 +26,35 @@
 
 namespace PrestaShopBundle\Form\Admin\Configure\AdvancedParameters\Administration;
 
+use PrestaShopBundle\Form\Admin\Type\MultistoreConfigurationType;
 use PrestaShopBundle\Form\Admin\Type\SwitchType;
 use PrestaShopBundle\Form\Admin\Type\TranslatorAwareType;
 use Symfony\Component\Form\FormBuilderInterface;
 use Symfony\Component\OptionsResolver\OptionsResolver;
 
 class NotificationsType extends TranslatorAwareType
 {
+    public const FIELD_SHOW_NOTIFS_NEW_ORDERS = 'show_notifs_new_orders';
+    public const FIELD_SHOW_NOTIFS_NEW_CUSTOMERS = 'show_notifs_new_customers';
+    public const FIELD_SHOW_NOTIFS_NEW_MESSAGES = 'show_notifs_new_messages';
```

Comment:
Just a suggestion, might be subjective, but I personally don't see the point shortening these :man_shrugging:
```suggestion
    public const FIELD_SHOW_NOTIFICATIONS_OF_NEW_ORDERS = 'show_notifs_new_orders';
    public const FIELD_SHOW_NOTIFICATIONS_OF_NEW_CUSTOMERS = 'show_notifs_new_customers';
    public const FIELD_SHOW_NOTIFICATIONS_OF_NEW_MESSAGES = 'show_notifs_new_messages';
```